### PR TITLE
Move imgproc python modules to siriuspy package

### PR DESCRIPTION
**Pull request by xresende**
related to https://github.com/lnls-sirius/machine-applications/pull/285

### CODE REVIEW

File: siriuspy/siriuspy/imgproc/main.py [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+
+from pcaspy import Alarm as _Alarm
+from pcaspy import Severity as _Severity
+
+from .. import util as _util
+
+from .meas import Measurement
+
+
+class App:
```
Comment (fernandohds564) [inline siriuspy/siriuspy/imgproc/main.py:14]: Please, change this name to something more explanatory.

Comment (xresende) [reply siriuspy/siriuspy/imgproc/main.py:14]: `meas`  module, `Measurement` class or `App` class? which one you mean. all are very generic.

I believe @RafaelLyra8  decided on these names based on what he saw in siriuspy `meas` subpackage.
But maybe rename `Measurement` class to something like `MeasBeamParams`  ou simply `MeasDVF`.

which name do you guys prefer?

Comment (fernandohds564) [reply siriuspy/siriuspy/imgproc/main.py:14]: I was referring to the `App` class. The name `Measurement` doesn't bother me. But, from the suggestions you gave, I prefer `MeasDVF`.

**Review by RafaelLyra8 — APPROVED** [review]

File: siriuspy/siriuspy/__init__.py [hunk 2]
```diff
@@ -12,4 +12,5 @@
     'cycle', 'devices', 'diagbeam', 'diagsys', 'epics', 'machshift',
     'magnet', 'meas', 'namesys', 'optics', 'opticscorr', 'oscilloscope',
     'posang', 'pwrsupply', 'ramp', 'search', 'simul', 'sofb', 'timesys',
+    'imgproc',
```
Comment (anacso17) [inline siriuspy/siriuspy/__init__.py:15]: maybe `blimgproc` or `dvfimgproc`?

File: siriuspy/siriuspy/imgproc/csdev.py [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+from mathphys.functions import get_namedtuple as _get_namedtuple
+
+from .. import util as _util
+from .. import csdev as _csdev
+from ..envars import VACA_PREFIX as _vaca_prefix
+from ..devices import DVF as _DVF
+
+
+# TODO: move this module to siriuspy
```
Comment (anacso17) [inline siriuspy/siriuspy/imgproc/csdev.py:13]: this comment can be removed :)

File: siriuspy/siriuspy/imgproc/main.py (hunk 1 above)
Comment (xresende) [reply siriuspy/siriuspy/imgproc/main.py:14]: @fernandohds564 , many IOCs define this `App` class as it is nowadays (DiagSys/fofb/idff/imgproc/injctrl/machshift/posang/stabinfo). It is more likely we break "standard" by changing App in imgproc than keeping as it is. I suggest we leave this renaming of all `App`classes for a separate PR.

File: siriuspy/siriuspy/imgproc/csdev.py (hunk 3 above)
Comment (xresende) [reply siriuspy/siriuspy/imgproc/csdev.py:13]: done

File: siriuspy/siriuspy/__init__.py (hunk 2 above)
Comment (xresende) [reply siriuspy/siriuspy/__init__.py:15]: done

**Review by anacso17 — APPROVED** [review]

File: siriuspy/siriuspy/imgproc/main.py (hunk 1 above)
Comment (fernandohds564) [reply siriuspy/siriuspy/imgproc/main.py]: Ok then.

**Review by fernandohds564 — APPROVED** [review]

**Review by Gabrielrezende-asc — APPROVED** [review]

**Pull request merged**